documentation/asciidoc/topics/proc_disabling_gossip_router_cross_site.adoc

```diff
@@ -0,0 +1,46 @@
+[id='disabling-gossip-router-cross-site_{context}']
+= Disabling local Gossip router and service
+
+[role="_abstract"]
+The {ispn_operator} starts a Gossip router on each site, but you only need a single Gossip router to manage traffic between the {brandname} cluster members.
+You can disable the additional Gossip routers to save resources.
+
+.Prerequisites
+
+* Have multiple working Gossip routers.
++
+You can have an external Gossip router that is not managed by the {ispn_operator}.
+
+.Procedure
+
+. Create an `Infinispan` CR for each {brandname} cluster.
+. Specify the name of the local site with `spec.service.sites.local.name`.
+. Set `false` as the value for the `spec.service.sites.local.discovery.launchGossipRouter`.
+. Connect to a remote site:
+.. For the remote site with Gossip router enabled,specify the `url` with the `spec.service.sites.locations.url`.
++
+[source,yaml,options="nowrap",subs=attributes+]
+----
+include::yaml/xsite_disable_gossip_router.yaml[]
+----
+.. For the remote site with Gossip router disabled, do not specify `spec.service.sites.locations.url`.
++
+[source,yaml,options="nowrap",subs=attributes+]
+----
+include::yaml/xsite_local_gossip_router.yaml[]
+----
++
+[IMPORTANT]
+====
+{brandname} recommends to keep the Gossip router enabled on all the remote sites.
```

Comment:
This is only true when 3 or more sites exist. If only 2 sites exist, then disabling the router is valid. This is because if there's two sites and a router goes down permanently, then connection between the two sites probably can't be achieved anyway.

Reply:
This makes sense Ryan. Thanks for the explanation. Updated